engineering/conventions.md

```diff
@@ -53,4 +54,11 @@ Refer to the [licensing](licensing.md), [maintainers](maintainers.md) and [docum
 
 ## Development Environments
 
-Each sourcerer has his own preferences on editors and IDEs. But if you do not know what to use, [Visual Studio Code](https://code.visualstudio.com/) is a popular choice for Go and Python, and [IntelliJ IDEA](https://www.jetbrains.com/idea/) is the most used for Java and Scala.
\ No newline at end of file
+Each sourcerer has his own preferences on editors and IDEs. But if you do not know what to use, [Visual Studio Code](https://code.visualstudio.com/) is a popular choice for Go and Python, and [IntelliJ IDEA](https://www.jetbrains.com/idea/) is the most used for Java and Scala.
+
+## Configuration
+
+Your application should be [configured](https://12factor.net/config) using environment variables. External services (e.g. databases) URLs, storage paths, [ports to bind](https://12factor.net/port-binding) and logging format all belong to configuration.
```

Comment:
So, if we have a lot of small configuration params, really common on databases to tweak them for specific use cases, are we going to put all of them using environment variables?

Reply:
Yes, we definitely should do that. Otherwise it is painful to pass config files to docker. In k8s you need to create a config map and then pass it to the pod to mount it, which is feasible, but it is a bit painful

Reply:
@ajnavarro In that case it's a good idea to allow connection string style.
For example:
`DATABASE=postgres://user:pass@host:port/database?sslmode=verify-full`

Reply:
Actually, not only in that case, it's a good idea most of the time. For example:
`BROKER=amqp://host:port` or `BROKER=memory://`.